lib/index.js

```diff
@@ -1155,12 +1155,15 @@ module.exports = (function() {
             }
         })
         if(imageInMergeCell == false){
-            var ratio = 1;
+            var ratio = 100;
```

Comment:
I see that you change meaning of parameter from scale to ratio in percents. Are this is intended? I like that change though.

Reply:
Yes, this is intended. I prefer this approach too.
The ratio cannot be equal or less than 0%. But it can be more than 100%, so, we can enlarge image. The quality may be worst but the developer/user manage what he wants